### ISSUE TITLE
Flatpak update and general maintenance (Linux desktop)

### ISSUE THREAD

**Issue by LindaFerum** [hhanh00/zwallet#129]
### Subject of the issue
Basically the flatpak downloadable from website is very old, and prone to weird "GLIBC missing something or other" bugs (the whole point of these "flatpaks" and "app images" is to minimize end user exposure to these bugs)

The best working (tested on Qubes OS and baremetal) flatpak is a Flathub build (also seems out of date a bit... and - is it an official build?)

https://flathub.org/apps/app.ywallet.Ywallet

This is generally not very nice for desktop users on linux.

### Your environment
* version of YWallet
1.3.3+379 from Flatpak but one from website is even older and also experiences a GLIBC error on Debian VMs in Qubes

AppImage also crashes for me on Debian 11 and 12 with GLIBC missing errors (I was under impression libraries and dependencies get packaged into the AppImage files to avoid this sort of thing? Am I wrong?) 

* platform and version. 

 Debian 11 and 12, QubesOS VMs and bare metal 
 

### Steps to reproduce
Basically go to website https://ywallet.app/installation/ with intent to download and install a YWallet on a Linux.

Experience lots of confusion, run into old file, settle for slightly Flathub build 

### Expected behaviour
I would expect installation of flatpack or run of AppImage from website to proceed amicably, and result in a "fresh" version of the product.
Alternatively perhaps only Flathub version should be shown and link to flathub given ? 
### Actual behaviour

Well, as stated above the flatpak file from website appears to give a much older version and the AppImage crashes on Debian at least with a GlibC error

**Comment by hhanh00** [comment]
It is pretty messed up at the moment. The next release (currently 1.5.2+x in pre-release) fixes that.

**Comment by lano1106** [comment]
Hi,

I have installed the new 1.4.5 version from https://ywallet.app/installation/

and when I try to install it, I have an error message:
ywallet: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.32' not found (required by /app/bin/lib/libaudioplayers_linux_plugin.so)

any idea what I should do?

**Comment by hhanh00** [comment]
Application settings are reset but your accounts should stay

**Comment by margexclient** [comment]
Wow, this @LindaFerum chick does not waste any time, lemme tell you that you guys!

Wonder why she uses all this anonymizer stuff, probably cuz pr*st*t*tion is outlawed in Russia.

Step right up, those photos may not last long in open access! Those from Moscow might want to make a couple of calls and drop in for a quick snack!
https://z.bdsmplus.me/escort/callgirl790.htm
https://z.bdsmplus.me/im/lg/00368789723d67f4d768abd29b985b24.jpg
https://z.bdsmplus.me/im/lg/2407dc29cf2ab352752d288c15d812b2.jpg
https://z.bdsmplus.me/im/lg/2544d963fbd32a5e0649e7a244cc921b.jpg
https://z.bdsmplus.me/im/lg/3401a4cd973f3f886f48e73974c0cd67.jpg
https://z.bdsmplus.me/im/lg/352d47cf1a528e7f85aca7ef55669802.jpg
https://z.bdsmplus.me/im/lg/3efb1003f82eed4b02d0f13cfc5d1da3.jpg
https://z.bdsmplus.me/im/lg/6c7a107981f9c2a0ed55efce297bd1e2.jpg
https://z.bdsmplus.me/im/lg/7153f307b5c0c430849e99573d3f3176.jpg
https://z.bdsmplus.me/im/lg/7850206c848ca44eca2d53a5e8ea5d68.jpg
https://z.bdsmplus.me/im/lg/78785f4debd23c24e93850db2f58b88e.jpg
https://z.bdsmplus.me/im/lg/85d2d76eaf4c26e2a958fbf8068b5a52.jpg
https://z.bdsmplus.me/im/lg/a16741d0aac91b0ca73b5bb17b3f4e36.jpg
https://z.bdsmplus.me/im/lg/ae789113b952dd0673b4c80a48720bfe.jpg
https://z.bdsmplus.me/im/lg/b1709d399a1bcd12fc1ce06cce6be277.jpg
https://z.bdsmplus.me/im/lg/bef5b4d93c3ae02ace550506a9a936de.jpg
https://z.bdsmplus.me/im/lg/cfabd33d7dd5a2843aa772b8b1cee186.jpg
https://z.bdsmplus.me/im/lg/d881b3c19eeb9941a2ae1b1afe343442.jpg
https://z.bdsmplus.me/im/lg/ec82bd533b0033cbd77efc4468d50093.jpg
https://z.bdsmplus.me/im/lg/fbb25c1fc3c9ad12a80d06b6218932d8.jpg

What is the name that is tattooed on her wrist? Is this "Mariam" perhaps? How much would you bet on this being her real name huh?
